config/test/manager_image_patch.yaml

```diff
@@ -10,4 +10,4 @@ spec:
       - name: manager
         args:
           - "--v=debug"
-          - "--wait-time=10"
+          - "--wait-time=600"
```

Comment:
Do you kno if we can say to kustomize "Add debug flag" instead of rewritting args ? so we don't have to specify wait-time here.

Reply:
I briefly google  it and didn't find anything of the kind..

Reply:
I think we should leave config/test  w/o `wait-time` argument and change the default value in main.go to be 60
so we could iterate faster on test suite.